### DISCUSSION TITLE
Multiple photo edit

### DISCUSSION THREAD

**Discussion by lleibovici** [photoprism/photoprism#1021] [Ideas]
It would be great to be able to edit a data field in multiple photos simutaniously.
For example after importing a lot of photos I found some without any date. It would be nice to be able to select several and apply the same year to all of them at onece. Currently I can select several but have to edit each one seperately and then go on to the next.

**Comment by graciousgrey** [comment]
This will be part of batch edit https://github.com/photoprism/photoprism/issues/271 :)
